client/components/web-preview/content.jsx

```diff
@@ -390,6 +391,7 @@ export default class WebPreviewContent extends Component {
 					isLoading={ this.state.isLoadingSubpage }
 					isSticky={ this.props.isStickyToolbar }
 				/>
+				<DomainUpsellCalout trackEvent="site_preview_domain_upsell_calout" />
```

Comment:
Will need to fix this from calout to callout.

Reply:
Fixed, I used the component we use on the Post Editor